src/In.ProjectEKA.HipService/DataFlow/ErrorMessage.cs

```diff
@@ -2,7 +2,10 @@ namespace In.ProjectEKA.HipService.DataFlow
 {
     public static class ErrorMessage
     {
-        public static readonly string InternalServerError = "Internal Server Error";
-        public static readonly string ContextArtefactIdNotFound = "Consent artefact not found.";
+        public const string InternalServerError = "Internal Server Error";
+        public const string ContextArtefactIdNotFound = "Consent artefact not found";
+        public const string InvalidToken = "Token is invalid";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5067092)